pyproject.toml

```diff
@@ -31,6 +31,8 @@ pep8-naming = "^0.12.0"
 types-toml = "^0.1.5"
 Pillow = "^8.3.1"
 azure-storage-file-share = "12.4.0"
+libvirt-python = {version = "^7.10.0", platform = 'linux'}
```

Comment:
What happens on Windows? It needs to keep Windows can run with other LISA platforms.

Reply:
Yeah, this is something I need to fix. I need to figure out a way to not load the QEMU orchestrator on Windows platforms.

Reply:
you may refer to `lisa\runners\legacy_runner.py`. It imports some modules in Windows only. You may put a mockup class for libvirt to pass type checking with the similar way.